Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -9,26 +9,30 @@
     <!-- XHED: Version title for the release info screen -->
     <string name="release_info_version_title">Release %1$s</string>
     <!-- XTXT: Description for the release info screen -->
-    <string name="release_info_version_body">Mit diesem Update beheben wir Fehler in der App. Es enthält keine neuen Funktionen.</string>
+    <string name="release_info_version_body">Mit diesem Update stellen wir Ihnen neben Fehlerbehebungen auch neue und erweiterte Funktionen zur Verfügung.</string>
     <!-- XBUT: Continue button for the release info screen -->
     <string name="release_info_continue_button">Weiter</string>
     <!-- XTXT: New release info footer -->
     <string name="new_release_bottom">Änderungen zum Release finden Sie in den App-Informationen unter dem Menüpunkt „Neue Funktionen”.</string>
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
+        <item>Verhaltensempfehlungen aktualisieren</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
+        <item>Die Verhaltensempfehlungen bei erhöhten Risiko wurden aktualisiert. Das betrifft unter anderem Empfehlungen zur Reduktion von Kontakten oder dem Tragen von Masken. Angepasst wurden auch die Empfehlungen bei niedrigem Risiko.</item>
     </string-array>
```

Comment:
....bei erhöhte**m** Risiko. ... would be correct German, however it is wrong in Figma

Reply:
Changed